src/components/CodeView/index.tsx

```diff
@@ -101,7 +103,7 @@ export class CodeViewBase extends React.Component<Props> {
     let codeWasTrimmed = false;
     let slowAlert;
 
-    if (content.length >= _slowLoadingCharCount) {
+    if (content.length >= _slowLoadingCharCount || isMinified) {
```

Comment:
Yeah, it's a bit unclear. Lots of similar names, probably because this evolved over time. `shouldAllowSlowPages` takes into account a URL parameter that we use for the user to be able to control whether files are truncated or not. This allows them to show the full file, even if the app thinks it should be truncated. `shouldAllowSlowPages` also allows us to configure what the default is, if no URL parameter has been provided. This is to allow the Compare view to default to no truncation.

`allowSlowPages`, farther down, is the actual value of the URL parameter in question.

These don't directly relate to the conditions `content.length >= _slowLoadingCharCount` and `isMinified`. Those are used as an initial gate to let the app know whether the content should be truncated at all or not. So if neither of those conditions is `true`, then we don't care about `shouldAllowSlowPages` or `allowSlowPages` because we are never going to truncate the file.

I hope that sheds some light on all of this.